--- a/internal/app/service.go	2026-08-23 11:26:08.000000000 +0000
+++ b/internal/app/service.go	2026-10-01 11:21:37.157485067 +0000
@@ -84,9 +84,12 @@
 
 		winner, isOver := domain.IsGameOver(currBoard)
 		if isOver {
-			if winner == 0 {
+			switch winner {
+			case 0:
 				currGame.Status = domain.StatusDraw
-			} else {
+			case 2:
+				currGame.Status = domain.StatusBotWon
+			default:
 				currGame.Status = domain.StatusPlayerWon
 			}
 			s.repository.Save(currGame)
--- a/internal/domain/model.go	2026-08-22 23:23:58.000000000 +0000
+++ b/internal/domain/model.go	2026-10-01 11:21:37.157248225 +0000
@@ -32,6 +32,7 @@
 	StatusPlayerMove       GameStatus = "Waiting for player move"
 	StatusWaitingForPlayer GameStatus = "Waiting for player connection"
 	StatusPlayerWon        GameStatus = "Player won"
+	StatusBotWon           GameStatus = "Bot won"
 	StatusDraw             GameStatus = "Draw"
 )
 
--- a/internal/store/mapper.go	2026-10-01 10:45:11.872704111 +0000
+++ b/internal/store/mapper.go	2026-10-01 11:21:37.157644907 +0000
@@ -25,6 +25,8 @@
 		storeGame.status = "DRAW"
 	case domain.StatusPlayerWon:
 		storeGame.status = "PLAYER_WON"
+	case domain.StatusBotWon:
+		storeGame.status = "BOT_WON"
 	default:
 		storeGame.status = "UNKNOWN"
 	}
@@ -60,6 +62,8 @@
 		domainGame.Status = domain.StatusDraw
 	case "PLAYER_WON":
 		domainGame.Status = domain.StatusPlayerWon
+	case "BOT_WON":
+		domainGame.Status = domain.StatusBotWon
 	default:
 		domainGame.Status = "unknown"
 	}
--- a/internal/store/storage.go	2026-10-01 10:43:09.680839405 +0000
+++ b/internal/store/storage.go	2026-10-01 11:21:37.157775328 +0000
@@ -238,7 +238,7 @@
 	query := `SELECT id, board, status, type, move_player_id, owner_player_id, created_at
 		FROM games
 		JOIN users_games on games.id = users_games.game_id
-		WHERE (status = 'PLAYER_WON' OR status = 'DRAW') and user_id = $1`
+		WHERE status IN ('PLAYER_WON', 'BOT_WON', 'DRAW') and user_id = $1`
 
 	res, err := s.pool.Query(ctx, query, userId)
 	games, err := pgx.CollectRows(res, pgx.RowToStructByNameLax[gameRow])
